@uportal/content-carousel/src/lib/AuthenticatedRssStrategy.ts

```diff
@@ -0,0 +1,59 @@
+import {CarouselItem} from '@/lib/CarouselItem';
+import {DataStrategy} from '@/lib/DataStrategy';
+import {parseXml} from '@/lib/parse';
+import oidc from '@uportal/open-id-connect';
+
+export class AuthenticatedRssStrategy implements DataStrategy {
+  public readonly type = 'Authenticated RSS';
+
+  public items: CarouselItem[] = [];
+
+  constructor(public feed: string) {
+    this.load(feed);
+  }
+
+  private async load(path: string): Promise<any> {
+    const token = (await oidc()).encoded;
+    const response = await fetch(path, {
+      credentials: 'same-origin',
+      headers: {
+        'Authorization': 'Bearer ' + token,
+        'content-type': 'application/jwt',
```

Comment:
I'm not sure using a `content-type` header here makes sense.

I think `content-type` is much more common in responses;  in a request (like here), I think it's meant to tell the server about the _body_ of the request.  You are sending a JWT, but it's part of the `Authorization` header.